app/templates/sectionsummary.html

```diff
@@ -0,0 +1,41 @@
+{% import 'macros/helpers.html' as helpers %}
+
+{% extends theme('section_confirmation.html') -%}
+
+{% block page_title -%}{{_("Section-Summary")}} - {{survey_title}}{% endblock -%}
+
+{% block sidebar %}
+  {% if navigation %}
+    {% include theme('partials/navigation.html') %}
+  {% endif %}
+{% endblock %}
+
+{% block form_content %}
+{% endblock %}
+
+{% block main -%}
+
+<div>
+    {% set group = content.summary.groups | first %}
+    {% for group in content.summary.groups %}
```

Comment:
Im not sure what these 2 lines do?
you assign group in the 1st then you reassign it in the 2nd